src/App.css

```diff
@@ -1,62 +1,72 @@
+* {
```

Comment:
the styles in this file will be global, if we need to make it to a specific component, we need to use CSS modules instead.